app/controllers/api/base_controller.rb

```diff
@@ -65,6 +67,28 @@ def response_with_mfa_warning(response)
     message
   end
 
+  def render_error_if_user_mfa_requirement_unmet(user)
+    if user.mfa_required_not_yet_enabled?
+      message = <<~ERROR.chomp
+        [ERROR] For protection of your account and your gems, you are required to set up multi-factor authentication \
```

Comment:
The errors returned in RubyGems doesn't have the `[ERROR]` prefix, I would personally remove it for consistency.